.github/workflows/build-and-deploy-staging.yml

```
@@ -1,21 +1,17 @@
-name: Azure Static Web Apps CI/CD
+name: Staging Checks and Deploy
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)